### PR TITLE
Add Parameter & Return Types (versions 3 of the library)

### PR DESCRIPTION
**Pull request by navarr**
Please see php-fig/fig-standards#1252 for an explanation

### CODE REVIEW

File: src/CacheInterface.php [hunk 1]
```diff
@@ -64,7 +64,7 @@ public function clear();
      *   MUST be thrown if $keys is neither an array nor a Traversable,
      *   or if any of the $keys are not a legal value.
      */
-    public function getMultiple($keys, $default = null);
+    public function getMultiple(iterable $keys, mixed $default = null): iterable;
```
Comment (Crell) [inline src/CacheInterface.php:67]: (Can't comment on the correct line above this; thanks GitHub...)

Should the docblock be updated to say `iterable<string>`, or whatever the de facto convention is for that?  The code cannot be, yet, but the docblock could.  I don't know if that's scope creep or not, but worth considering.  (Here and in a few other methods.)

**Comment by navarr** [comment]
I've force-pushed.  This rebases the iterable document types to be the first commit, so that all proposed releases include them.

**Comment by navarr** [comment]
I've force pushed.  This rebase updates the commit messages and the composer.json files to align with recent edits to the meta document

**Comment by Geolim4** [comment]
Can you mention me once this MR is merged please ?

I need to prepare [the next major update of Phpfastcache](https://github.com/PHPSocialNetwork/phpfastcache/tree/v9), thanks 👼🏻

**Comment by Crell** [comment]
Everything looks good here, I think.  Just some issues on the spec PR itself.

**Comment by Geolim4** [comment]
Hello @Crell, not wanting to be pressing or anything but  this seems to take much longer than it took for the PSR-6 to make the PHP-8 type upgrade ?

Thanks again :)

**Comment by Crell** [comment]
Someone needs to call the vote on it.  Paging @navarr or @Jean85...

**Comment by Jean85** [comment]
I'm not ok with having this in a single PR before calling a vote. We need two separate PR/branches, to have a clear separation of the proposed tags.

**Comment by Geolim4** [comment]
> I'm not ok with having this in a single PR before calling a vote. We need two separate PR/branches, to have a clear separation of the proposed tags.

Can the concerned people make the changes and proceed to the vote ? This "simple" upgrade for PHP 8 is during for almost five months now.

It's in the FIG's interest to be proactive and do the good things to improve interoperability for PHP community.

Thank you @Jean85 🙏🏻

**Comment by Jean85** [comment]
I've done it myself, by splitting this into #25 and changing the base here.

I'll try to call a vote ASAP.

**Comment by dragoonis** [comment]
As long as things are addressed, as per 
https://github.com/php-fig/simple-cache/pull/13#issuecomment-616652917
then we are good here.

**Comment by navarr** [comment]
@Jean85 Why are the PRs split?  Do we need two separate votes for these changes?

**Comment by Jean85** [comment]
> @Jean85 Why are the PRs split? Do we need two separate votes for these changes?

No but we will need two different releases, so we need to make the CC able to review those separately. And us to tag separately, too.

**Comment by Jean85** [comment]
Closed by mistake (stupid automations!)

Vote has passed: https://groups.google.com/g/php-fig/c/DJ8ysB1JV4Y
PSR changes are merged: https://github.com/php-fig/fig-standards/pull/1252

Releasing this as 3.0.0 now.

**Comment by Geolim4** [comment]
Yes, finally 🥳 🎉 

Thanks for that and sorry for pushing again and again, I will have some work this weekend :D

**Comment by Geolim4** [comment]
@Jean85 can you create the Github releases after pushing tags too ? Thank you !

**Comment by Jean85** [comment]
@Geolim4 done!
